python/sparkdl/udf/keras_image_model.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+
+from ..graph.builder import GraphFunction, IsolatedSession
+from ..graph.pieces import buildSpImageConverter, buildFlattener
+from ..image.imageIO import imageSchema
+from ..utils import jvmapi as JVMAPI
+
+logger = logging.getLogger('sparkdl')
+
+def _serialize_and_reload_with(preprocessor):
+    """
+    Load a preprocessor function (image_file_path => image_tensor)
```

Comment:
More like "Returns a function that loads and preprocesses images using the given preprocessor  (image_file_path => image_tensor).